libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -935,13 +935,14 @@ void AP_OSD_Screen::draw_current(uint8_t x, uint8_t y)
     AP_BattMonitor &battery = AP::battery();
     float amps;
     if (!battery.current_amps(amps)) {
-        amps = 0;
+        osd->avg_current_a = 0;
     }
-    if (amps < 10.0) {
-        backend->write(x, y, false, "%2.2f%c", amps, SYM_AMP);
+    osd->avg_current_a= osd->avg_current_a + (amps - osd->avg_current_a)/3;
```

Comment:
use * 0.33 instead of /3, as multiply is faster than division
also, a comment would be nice :-)

Reply:
done....thanks